src/scripts/dungeons/Dungeon.ts

```diff
@@ -2692,11 +2680,98 @@ dungeonList['Solaceon Ruins'] = new Dungeon('Solaceon Ruins',
+        }),
+        new DungeonBossPokemon('Unown (K)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 8),
+        }),
+        new DungeonBossPokemon('Unown (L)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 9),
+        }),
+        new DungeonBossPokemon('Unown (M)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 10),
+        }),
+        new DungeonBossPokemon('Unown (N)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 11),
+        }),
+        new DungeonBossPokemon('Unown (O)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 12),
+        }),
+        new DungeonBossPokemon('Unown (Q)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 13),
+        }),
+        new DungeonBossPokemon('Unown (R)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 14),
+        }),
+        new DungeonBossPokemon('Unown (S)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 15),
+        }),
+        new DungeonBossPokemon('Unown (T)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 16),
+        }),
+        new DungeonBossPokemon('Unown (U)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 17),
+        }),
+        new DungeonBossPokemon('Unown (V)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 18),
+        }),
+        new DungeonBossPokemon('Unown (W)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 19),
+        }),
+        new DungeonBossPokemon('Unown (X)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 20),
+        }),
+        new DungeonBossPokemon('Unown (Y)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 21),
+        }),
+        new DungeonBossPokemon('Unown (Z)', 4100000, 30, {
+            hide: true,
+            requirement: new SeededDateRequirement(() => SeededDateRand.intBetween(0, 22) == 22),
+        }),
```

Comment:
I was originally going to add something like this to this dungeon too, but the main issue is that it will be the same encounters (for most days).
The only thing I could think that might help, is adjusting the order?
Would still make each dungeons encounters basically tied together, but would at least be a different Unown in each dungeon on each day.

Reply:
Actually I think my theory that the cycles are out of sync because there's different amounts of Unowns holds up. Like, if I test it today there's different Unowns.

Reply:
It'll always be similar though,
Like if you set the numbers to 0 → 100 and it selected 50 today, then you set the numbers to 0 → 50 it will be 25.

Reply:
How 'bout now?

Reply:
We could make them use different seeds. Like the first one uses CurrentDate, while the other one uses CurrentDate + 3. Should give a random enough pattern, while still maintaining the changes-each-day-feature.
It is probably out side Ultima's coding skills, but i can look at it, in some days, unless Red wants to

Reply:
> It is probably out side Ultima's coding skills

Oi. Rude.

But true.